lib/tasks/default.rake

```diff
@@ -373,6 +373,26 @@ task :save_en do
   sh 'cp -p config/locales/en.yml config/locales/en.yml.ORIG'
 end
 
+desc 'Test locale translation'
+task :test_locale do
+  # Use 'system' so we return the error code (error=0 is okay)
+  system 'rails test test/models/translations_test.rb'
+end
+
+desc 'Fix locale text'
+task :fix_locale do
+  # Google Translate generates text that has predictable errors; here
+  # are ways to fix them automatically.  Commented out, instead of being
+  # removed entirely, so that we can quickly restore them in a pinch.
+  sh %q{ruby -pi -e "sub(/< a /, '<a ')" } + files
+  sh %q{ruby -pi -e "sub(/< \057/, '</')" } + files
+  sh %q{ruby -pi -e "sub(/<\057 /, '</')" } + files
+  sh %q{ruby -pi -e "sub(/<Strong>/, '<strong>')" } + files
+  sh %q{ruby -pi -e "sub(/<Em>/, '<em>')" } + files
+  sh %q{ruby -pi -e "sub(/href = /, 'href=')" } + files
+  Rake::Task['test_locale'].invoke
```

Comment:
This is fine, but you're taking on some unnecessary overhead by invoking 6 different shells and loading the Ruby interpreter 6 times. Instead, you can chain the `sub`s together and call them on files a single time.